index.html

```diff
@@ -49,7 +49,15 @@
         color: rgb(200, 200, 200);
       }
     </style>
-    <title>Start typing!</title>
+    <title>Big Typer</title>
```

Comment:
A real title is super important. This will still change after typing, but on initial load this is a better title for search ranking.